kiran.bash

```diff
@@ -0,0 +1,7 @@
+mkdir code
+mkdir data
+mkdir data/raw_data
+mkdir output
+mkdir output/figures
+mkdir output/tables
+curl -L http://npk.io/aMPCOe+ -o data/raw_data/filename.fasta
```

Comment:
Also, the intention was to change the name of the downloaded file to something more useful than `filename.fasta`.